subsys/bluetooth/audio/ascs.c

```diff
@@ -1312,7 +1312,10 @@ static ssize_t ascs_config(struct bt_ascs *ascs, struct net_buf_simple *buf)
 
 	BT_DBG("num_ases %u", req->num_ases);
 
-	if (buf->len < req->num_ases * sizeof(*cfg)) {
+	if (req->num_ases < 1) {
+		BT_ERR("Number_of_ASEs parameter value is less than 1");
```

Comment:
I realize that the issue already existed in this file, but BT_ERR() should really be reserved for local errors only. Whenever a remote peer sends something invalid that we cleanly handle then BT_WARN() is the right choice.